bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -161,6 +163,59 @@ spec:
                   sleep 2
                   done
                 fi
+
+                election_timer="${OVN_NB_RAFT_ELECTION_TIMER}"
```

Comment:
Does this affect ovnkube-master? Would it make sense for it to be a separate container? It could just apply changes as necessary.

Reply:
I think setting this timer while the master pods startup will give us bit more deterministic behavior. If user start cluster with high number of worker nodes, we can get into the death loop of raft partition. 
Also i believe the general guideline is that we don't want to allow user to change this bahavior dynamically.

Reply:
Right, what I'm saying is that this should be a separate container in the master pod, rather than gating the master container.

Reply:
If we think of allowing user to change this value dynamically, i think having separate container makes sense to me. But if it's one time startup config, there is nothing much for this container to do, it will come up, set the value and die, and not sure CNO will be happy with that.